numba/cuda/cudadrv/driver.py

```diff
@@ -35,11 +35,9 @@
 from .drvapi import API_PROTOTYPES
 from .drvapi import cu_occupancy_b2d_size, cu_stream_callback_pyobj, cu_uuid
 from numba.cuda.cudadrv import enums, drvapi, _extras
-from numba.cuda.envvars import get_numba_envvar
 
 
-VERBOSE_JIT_LOG = int(get_numba_envvar('VERBOSE_CU_JIT_LOG', 1))
-MIN_REQUIRED_CC = (2, 0)
+MIN_REQUIRED_CC = (3, 0)
```

Comment:
Whilst unlikely, is this going to break anything?

Reply:
I don't think so - Nothing pre-3.0 is supported by any CUDA supported toolkit version anyway.

Reply:
Thanks for confirming.